### DISCUSSION TITLE
Can the manipulation of the editing window be changed to the control mode of Unity or Maya, making it easier for map editors to use?

### DISCUSSION THREAD

**Discussion by lelzhang** [godotengine/godot-proposals#7648] [Editor]
Can the manipulation of the editing window be changed to the control mode of Unity or Maya, making it easier for map editors to use? If not, at least it can be customized in the editor to a control mode familiar to artists, making Godot more user-friendly.

**Comment by Zireael07** [comment]
There is Maya control scheme in editor settings.

**Reply by lelzhang** [reply]
yes，thanks
